modules/local/process/bowtie_align.nf

```diff
@@ -0,0 +1,46 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+def options    = initOptions(params.options)
+
+process BOWTIE_ALIGN {
+    tag "$meta.id"
+    label 'process_high'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:meta.id) }
+
+    // Note: 2.7X indices incompatible with AWS iGenomes.
```

Comment:
copy-paste error from the STAR process 🙂

Reply:
Oops :) a lot of copy-pasting was involved, I admit it